### PR TITLE
Created works and about me section

### PR DESCRIPTION
**Pull request by Xecyc**
Hey! In this code review I created the works section and the about-me section of the mobile version. Breakpoints for both the mobile and the desktop version have been included. What I've done:

- Works section aligned with grid and flexbox
- Added button interactions
- Used responsive units
- Implemented responsive web design
- Added breakpoints

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -50,5 +50,162 @@ <h1 class="head-h1">Hey there.<br>I'm Edu.<span><h2>I'm a software developer</h2
         </ul>
       </div>
     </section>
+    <section class="works-container" id="works">
+      <div class="works-header-container header-area">
+        <div class="works-header">
+          <h1>My recent works</h1>
```
Comment (Efiamotu-1) [inline index.html]: - [ ] The html standard is that you should use only one `h1` tag in your whole application. you can visit this [link](https://www.reliablesoft.net/h1-tag/) and watch this [video](https://www.youtube.com/watch?v=rH-4w_0F1vM) to learn more about this. you can use any `hx` tag such as `h1, h2` etc.

File: index.html [hunk 2]
```diff
@@ -50,5 +50,162 @@ <h1 class="head-h1">Hey there.<br>I'm Edu.<span><h2>I'm a software developer</h2
...
+          <div class="line"></div>
+        </div>
+        <div class="works-header-multipost">
+          <img class="img-placeholder" src="./img/placeholders/Img Placeholder.png" alt="placeholder-image">
+          <div class="works-stories">
+            <h1>Multi-Post Stories</h1>
```
Comment (Efiamotu-1) [inline index.html]: - [ ] The html standard is that you should use only one `h1` tag in your whole application. you can visit this [link](https://www.reliablesoft.net/h1-tag/) and watch this [video](https://www.youtube.com/watch?v=rH-4w_0F1vM) to learn more about this. you can use any `hx` tag such as `h1, h2` etc.

File: index.html [hunk 3]
```diff
@@ -50,5 +50,162 @@ <h1 class="head-h1">Hey there.<br>I'm Edu.<span><h2>I'm a software developer</h2
...
+                <a tabindex="0" href="#" class="btn-action btn-area btn">See Project</a>
+              </ul>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div class="project-box left" id="project-1">
+        <div class="project-content">
+          <div class="project-text">
+            <h1>Profesional Art<br>Printing Data</h1>
```
Comment (Efiamotu-1) [inline index.html]: - [ ] The html standard is that you should use only one `h1` tag in your whole application. you can visit this [link](https://www.reliablesoft.net/h1-tag/) and watch this [video](https://www.youtube.com/watch?v=rH-4w_0F1vM) to learn more about this. you can use any `hx` tag such as `h1, h2` etc.

File: index.html [hunk 4]
```diff
@@ -50,5 +50,162 @@ <h1 class="head-h1">Hey there.<br>I'm Edu.<span><h2>I'm a software developer</h2
...
+              <li class="list-area">Ruby</li>
+            </ul>
+          </div>
+          <a tabindex="0" class="btn-action large-btn-area btn" href="#">See Project</a>
+        </div>
+      </div>
+      <div class="project-box middle" id="project-2">
+        <div class="project-content">
+          <div class="project-text">
+            <h1>Profesional Art<br>Printing Data</h1>
```
Comment (Efiamotu-1) [inline index.html]: - [ ] The html standard is that you should use only one `h1` tag in your whole application. you can visit this [link](https://www.reliablesoft.net/h1-tag/) and watch this [video](https://www.youtube.com/watch?v=rH-4w_0F1vM) to learn more about this. you can use any `hx` tag such as `h1, h2` etc.

File: index.html [hunk 5]
```diff
@@ -50,5 +50,162 @@ <h1 class="head-h1">Hey there.<br>I'm Edu.<span><h2>I'm a software developer</h2
...
+              <li class="list-area">Ruby</li>
+            </ul>
+          </div>
+          <a tabindex="0" class="btn-action large-btn-area btn" href="#">See Project</a>
+        </div>
+      </div>
+      <div class="project-box right" id="project-3">
+        <div class="project-content">
+          <div class="project-text">
+            <h1>Profesional Art<br>Printing Data</h1>
```
Comment (Efiamotu-1) [inline index.html]: - [ ] The html standard is that you should use only one `h1` tag in your whole application. you can visit this [link](https://www.reliablesoft.net/h1-tag/) and watch this [video](https://www.youtube.com/watch?v=rH-4w_0F1vM) to learn more about this. you can use any `hx` tag such as `h1, h2` etc.

File: index.html [hunk 6]
```diff
@@ -50,5 +50,162 @@ <h1 class="head-h1">Hey there.<br>I'm Edu.<span><h2>I'm a software developer</h2
...
+              <li class="list-area">Ruby</li>
+            </ul>
+          </div>
+          <a tabindex="0" class="btn-action large-btn-area btn" href="#">See Project</a>
+        </div>
+      </div>
+      <div class="project-box" id="project-4">
+        <div class="project-content">
+          <div class="project-text">
+            <h1>Profesional Art<br>Printing Data</h1>
```
Comment (Efiamotu-1) [inline index.html]: - [ ] The html standard is that you should use only one `h1` tag in your whole application. you can visit this [link](https://www.reliablesoft.net/h1-tag/) and watch this [video](https://www.youtube.com/watch?v=rH-4w_0F1vM) to learn more about this. you can use any `hx` tag such as `h1, h2` etc.

File: index.html [hunk 7]
```diff
@@ -50,5 +50,162 @@ <h1 class="head-h1">Hey there.<br>I'm Edu.<span><h2>I'm a software developer</h2
...
+              <li class="list-area">Ruby</li>
+            </ul>
+          </div>
+          <a tabindex="0" class="btn-action large-btn-area btn" href="#">See Project</a>
+        </div>
+      </div>
+      <div class="project-box" id="project-5">
+        <div class="project-content">
+          <div class="project-text">
+            <h1>Profesional Art<br>Printing Data</h1>
```
Comment (Efiamotu-1) [inline index.html]: - [ ] The html standard is that you should use only one `h1` tag in your whole application. you can visit this [link](https://www.reliablesoft.net/h1-tag/) and watch this [video](https://www.youtube.com/watch?v=rH-4w_0F1vM) to learn more about this. you can use any `hx` tag such as `h1, h2` etc.

File: index.html [hunk 8]
```diff
@@ -50,5 +50,162 @@ <h1 class="head-h1">Hey there.<br>I'm Edu.<span><h2>I'm a software developer</h2
...
+              <li class="list-area">Ruby</li>
+            </ul>
+          </div>
+          <a tabindex="0" class="btn-action large-btn-area btn" href="#">See Project</a>
+        </div>
+      </div>
+      <div class="project-box" id="project-6">
+        <div class="project-content">
+          <div class="project-text">
+            <h1>Profesional Art<br>Printing Data</h1>
```
Comment (Efiamotu-1) [inline index.html]: - [ ] The html standard is that you should use only one `h1` tag in your whole application. you can visit this [link](https://www.reliablesoft.net/h1-tag/) and watch this [video](https://www.youtube.com/watch?v=rH-4w_0F1vM) to learn more about this. you can use any `hx` tag such as `h1, h2` etc.

File: index.html [hunk 9]
```diff
@@ -50,5 +50,162 @@ <h1 class="head-h1">Hey there.<br>I'm Edu.<span><h2>I'm a software developer</h2
...
+            </ul>
+          </div>
+          <a tabindex="0" class="btn-action large-btn-area btn" href="#">See Project</a>
+        </div>
+      </div>
+    </section>
+    <section class="about_me" id="about">
+      <div class="about_me-container">
+        <div class="about_me-text">
+          <h1>About me</h1>
```
Comment (Efiamotu-1) [inline index.html]: - [ ] The html standard is that you should use only one `h1` tag in your whole application. you can visit this [link](https://www.reliablesoft.net/h1-tag/) and watch this [video](https://www.youtube.com/watch?v=rH-4w_0F1vM) to learn more about this. you can use any `hx` tag such as `h1, h2` etc.

**Review by Efiamotu-1 — CHANGES_REQUESTED** [review]
Hi @Xecyc, 

 

Good job so far! 👏. Your Code is Awesome! 💪

There are some **few** issues 🐛 that you still need to work 🔧 on though to go to the next project ⏭️ but you are almost there! 🏃
## Don't Worry 🙏

It's ok to have bugs in your code 🐛. Please, see this code review as a **chance** for you to **learn** practical coding skills and learn **new tips** that can improve your code quality 💯 and help you **get a better ** 💼.

_I am going to guide you on how to fix them down below. 🤓_

## Highlights :dart: 
- ✔️ No linter Errors
- ✔️ Descriptive README
- ✔️ Descriptive PR title and description
- The code is top-notch! 💯 and has a huge potential.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

> **Note:** Please, don't Merge This Pull Request yet. Only Merge when it is Approved ✅.


## Thank you for your time 🙏 and call me if you need help or clarification ☎️

- This is My **GitHub**: [@Efiamotu-1](https://github.com/Efiamotu-1/)
- This is My **Twitter**: [@EFYAMOTU](https://twitter.com/EFYAMOTU)
- This is My **linkedIn**.[Habeeb Efiamotu Musa](https://www.linkedin.com/in/musa-habeeb/)
- This is My **Slack**: [Habeeb Efiamotu Musa](https://slack.com)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 10]
```diff
@@ -50,5 +50,162 @@ <h1 class="head-h1">Hey there.<br>I'm Edu.<span><h2>I'm a software developer</h2
         </ul>
       </div>
     </section>
+    <section class="works-container" id="works">
+      <div class="works-header-container header-area">
+        <div class="works-header">
+          <h2>My recent works</h2>
+          <div class="line"></div>
```
Comment (KanzaTahreem) [inline index.html:57]: - [OPTIONAL] Well done with the design 🥇🥇 What do you think of adding a bit more space here according to the Figma template? 🤔✨ Also, can you please use `font-weight: 400` for  **_My recent works_** ? 😄

Your design | Given design
---| ---|
image | image

File: index.html [hunk 11]
```diff
@@ -50,5 +50,162 @@ <h1 class="head-h1">Hey there.<br>I'm Edu.<span><h2>I'm a software developer</h2
...
+        </div>
+        <div class="works-header-multipost">
+          <img class="img-placeholder" src="./img/placeholders/Img Placeholder.png" alt="placeholder-image">
+          <div class="works-stories">
+            <h2>Multi-Post Stories</h2>
```
Comment (KanzaTahreem) [inline index.html:62]: - [OPTIONAL] Kindly consider aligning this heading to the start as given in the design guidelines. See the images below👇

Your design | Given design
---| ---|
image | image

File: index.html [hunk 12]
```diff
@@ -50,5 +50,162 @@ <h1 class="head-h1">Hey there.<br>I'm Edu.<span><h2>I'm a software developer</h2
...
+        </div>
+      </div>
+    </section>
+    <section class="about_me" id="about">
+      <div class="about_me-container">
+        <div class="about_me-text">
+          <h2>About me</h2>
+          <p>Hello I’m a software developer! I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+          <a tabindex="0" class="btn-action btn btn-area" href="#">Get my resume</a>
+        </div>
```
Comment (KanzaTahreem) [inline index.html:167]: - [OPTIONAL] Please consider adding a bit of gap between the elements of this `div` ⛏🤓 You can use `gap: 2rem or your preffered value` for `.about_me-text`. See the images below for clarity 💡

Your design | Given design
---| ---|
image | image

**Review by KanzaTahreem — APPROVED** [review]
# Approved 🥳

Hi @Xecyc 🙋‍♀️

Great work implementing the suggested changes 💪 **_Your project is COMPLETE! There is nothing else to say other than... it's time to merge it! 🚀 Congratulations!_**



## Highlights 🎯
There are some excellent points that you have made perfectly. Keep up the great work! 👏

-  ✅ Styling is looking great, perfect!
-  ✅ Great work using the right GitHub Flow 
-  ✅ Your README looks good, well done!
-  ✅ Linters are working properly, great job! 

## Optional suggestions ❕
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

_Please check the comments under review_

Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**